### PR TITLE
Add Boba Network

### PR DESCRIPTION
**Pull request by jet86**
Added Boba Network project and bridges.

### CODE REVIEW

File: src/helpers/bridge_contracts.js [hunk 1]
```diff
@@ -437,6 +447,21 @@ export const ETH_BRIDGE_CONTRACTS = {
     ],
     ...PROJECT_BINANCE_SMART_CHAIN,
   },
+  [PROJECT_BOBA_NETWORK.name]: {
+    "bridges": [
+      {
+        "address": "0xdc1664458d2f0B6090bEa60A8793A4E66c2F1c0",
```
Comment (gasolin) [inline src/helpers/bridge_contracts.js]: can't find such address in https://etherscan.io/search?f=0&q=0xdc1664458d2f0B6090bEa60A8793A4E66c2F1c0 please confirm

**Review by gasolin — CHANGES_REQUESTED** [review]
Congrats for shipping the new L2 network, and Thanks for the contribution!
Please help clarify if the first contract is valid

File: src/helpers/bridge_contracts.js (hunk 1 above)
Comment (jet86) [reply src/helpers/bridge_contracts.js]: Thanks for picking up on that - I'd somehow left the last character off the end of the address. I've fixed that now.

**Review by gasolin — APPROVED** [review]
thanks!

**Comment by gasolin** [comment]
https://ethhole.link/project/Boba%20Network

**Pull request merged**